[2022-08-03 15:46:56,418] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.load-data manual__2022-08-03T15:46:23.412822+00:00 [queued]>
[2022-08-03 15:46:56,483] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.load-data manual__2022-08-03T15:46:23.412822+00:00 [queued]>
[2022-08-03 15:46:56,491] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:46:56,505] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-03 15:46:56,512] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 15:46:56,570] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): load-data> on 2022-08-03 15:46:23.412822+00:00
[2022-08-03 15:46:56,610] {standard_task_runner.py:52} INFO - Started process 1213 to run task
[2022-08-03 15:46:56,633] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ltv-pipeline', 'load-data', 'manual__2022-08-03T15:46:23.412822+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/dag-test.py', '--cfg-path', '/tmp/tmpb3l8l27o', '--error-file', '/tmp/tmpq52jw_by']
[2022-08-03 15:46:56,663] {standard_task_runner.py:80} INFO - Job 49: Subtask load-data
[2022-08-03 15:46:56,962] {task_command.py:371} INFO - Running <TaskInstance: ltv-pipeline.load-data manual__2022-08-03T15:46:23.412822+00:00 [running]> on host 3b0f05b8acbc
[2022-08-03 15:46:57,193] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ltv-pipeline
AIRFLOW_CTX_TASK_ID=load-data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T15:46:23.412822+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-03T15:46:23.412822+00:00
[2022-08-03 15:46:57,237] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-08-03 15:46:57,264] {dbapi.py:231} INFO - Running statement: 
            COPY vehicles (id, vin, price, miles, stock_no, year, make, model, trim, body_type, vehicle_type, drivetrain, transmission, fuel_type, engine_size, engine_block, seller_name, street, city, state, zip)
            FROM PROGRAM 'gzip -dc /opt/***/files/vehicle_challenge.csv.gz' DELIMITER ',' CSV HEADER;, parameters: None
[2022-08-03 15:46:57,304] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 92, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 211, in run
    self._run_command(cur, sql_statement, parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/hooks/dbapi.py", line 235, in _run_command
    cur.execute(sql_statement)
psycopg2.errors.ExternalRoutineException: program "gzip -dc /opt/***/files/vehicle_challenge.csv.gz" failed
DETAIL:  child process exited with exit code 1

[2022-08-03 15:46:57,405] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=ltv-pipeline, task_id=load-data, execution_date=20220803T154623, start_date=20220803T154656, end_date=20220803T154657
[2022-08-03 15:46:57,457] {standard_task_runner.py:97} ERROR - Failed to execute job 49 for task load-data (program "gzip -dc /opt/***/files/vehicle_challenge.csv.gz" failed
DETAIL:  child process exited with exit code 1
; 1213)
[2022-08-03 15:46:57,507] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-08-03 15:46:57,677] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
